x-pack/legacy/plugins/siem/package.json

```diff
@@ -14,7 +14,6 @@
     "@types/js-yaml": "^3.12.1",
     "@types/lodash": "^4.14.110",
     "@types/react-beautiful-dnd": "^10.0.1",
-    "cypress": "^3.4.1",
     "js-yaml": "^3.13.1",
```

Comment:
Looking through this file and comparing this `package.json` to the `x-pack/package.json` I think you want to remove these duplicates as well:

```ts
"@types/js-yaml": "^3.12.1"
    "js-yaml": "^3.13.1",
    "mocha-junit-reporter": "^1.23.1",
    "mocha-multi-reporters": "^1.1.7",
    "mochawesome": "^4.0.1",
    "mochawesome-merge": "^2.0.1",
    "mochawesome-report-generator": "^4.0.1",
    "ts-loader": "^6.0.4"
```

And then remove this extra one:
```ts
@cypress/webpack-preprocessor
```
but move it up one to `x-pack/package.json` as that looks like it was missed at some point.

Reply:
moved `@cypress/webpack-preprocessor` and `@types/js-yaml` was already in x-pack/package.json.